enterprise/backend/test/metabase_enterprise/serialization/v2/e2e/yaml_test.clj

```diff
@@ -145,73 +167,89 @@
-            (is (= 110 (count (dir->file-set (io/file dump-dir "Collection"))))))
-
-          (testing "for Databases"
-            (is (= 10 (count (dir->file-set (io/file dump-dir "Database"))))))
-
-          (testing "for Tables"
-            (is (= 100
-                   (reduce + (for [db    (get @entities "Database")
-                                   :let [tables (dir->file-set (io/file dump-dir "Database" (:name db) "Table"))]]
-                               (count tables))))
-                "Tables are scattered, so the directories are harder to count"))
-
-          (testing "for Fields"
-            (is (= 1000
-                   (reduce + (for [db    (get @entities "Database")
-                                   table (subdirs (io/file dump-dir "Database" (:name db) "Table"))]
-                               (->> (io/file table "Field")
-                                    dir->file-set
-                                    count))))
-                "Fields are scattered, so the directories are harder to count"))
-
-          (testing "for cards"
-            (is (= 100 (count (dir->file-set (io/file dump-dir "Card"))))))
-
-          (testing "for dashboards"
-            (is (= 100 (count (dir->file-set (io/file dump-dir "Dashboard"))))))
-
-          (testing "for metrics"
-            (is (= 30 (count (dir->file-set (io/file dump-dir "Metric"))))))
-
-          (testing "for segments"
-            (is (= 30 (count (dir->file-set (io/file dump-dir "Segment"))))))
-
-          (testing "for pulses"
-            (is (= 30 (count (dir->file-set (io/file dump-dir "Pulse"))))))
-
-          (testing "for pulse cards"
-            (is (= 120 (reduce + (for [pulse (get @entities "Pulse")]
-                                   (->> (io/file dump-dir "Pulse" (:entity_id pulse) "PulseCard")
-                                        dir->file-set
-                                        count))))))
-
-          (testing "for pulse channels"
-            (is (= 30 (reduce + (for [pulse (get @entities "Pulse")]
-                                  (->> (io/file dump-dir "Pulse" (:entity_id pulse) "PulseChannel")
-                                       dir->file-set
-                                       count)))))
-            (is (= 40 (reduce + (for [{:keys [recipients]} (get @entities "PulseChannel")]
-                                  (count recipients))))))
-
-          (testing "for native query snippets"
-            (is (= 10 (count (dir->file-set (io/file dump-dir "NativeQuerySnippet"))))))
-
-          (testing "for timelines and events"
-            (is (= 10 (count (dir->file-set (io/file dump-dir "Timeline"))))))
-
-          (testing "for settings"
-            (is (.exists (io/file dump-dir "settings.yaml")))))
+            (is (= 110 (-> @entities (get "Collection") count))))
+
+          (testing "storage"
+            (storage.yaml/store! (seq @extraction) dump-dir)
+            (def files (file-set (io/file dump-dir)))
```

Comment:
This looks like another thing you didn't mean to commit

Reply:
Sort of. It should just be inlined into the one usage. Done.